prompt.go

```diff
@@ -1,16 +1,22 @@
 package prompt
 
-import "github.com/howeyc/gopass"
+import (
```

Comment:
Can you put one import on each line as previously (or group all the imports together) please? I know it's the original external commit but it looks really messy (and maybe the reason the PR was never accepted.